source/renderer/app/themes/utils/constants.js

```diff
@@ -30,3 +36,59 @@ export const CREATE_THEME_MOCK_PARAMS = {
   },
   fonts: DEFAULT_FONTS,
 };
+
+export const CREATE_THEME_OBJ = {
+  ...createReactPolymorphTheme(CREATE_THEME_MOCK_PARAMS),
+  ...createDaedalusComponentsTheme(CREATE_THEME_MOCK_PARAMS),
+};
+
```

Comment:
@MarcusHurney  Related to "automatism" I mentioned, here can be something like themeParams that you can pass  as  `[CREATE_CARDANO_THEME_PARAMS, CREATE_LIGHT_BLUE_THEME_PARAMS... ]`

Reply:
This `CREATE_THEME_OBJ` is only needed to compare its keys against the existing themes. The actual values of the keys aren’t important. Not sure if this ties into the array of actual themes params you’ve suggested. We can discuss it further though.